File: LeetCode-Problems/Easy/Defanging-IP-Address.java
//Java
class Solution {
    public String defangIPaddr(String address) {
        String returnedString = "";
        for (int i = 0; i < address.length(); i++){
            if (address.substring(i, i+1).equals(".")){
                returnedString += "[.]";
            }
            else{
                returnedString += address.substring(i,i+1);
            }
        }
        return returnedString;
    }
}

File: LeetCode-Problems/Easy/Defanging-IP-Address.py
# Python
class Solution(object):
    def defangIPaddr(self, address):
        """
        :type address: str
        :rtype: str
        """
        returnedStr = "";
        for x in range(0, len(address)):
            if address[x] == ".":
                returnedStr += ("[.]")
            else:
                returnedStr += (address[x])
        return returnedStr
